### ISSUE TITLE
PHP is not resumed when using built-in web server

### ISSUE THREAD

**Issue by ghost** [bobthecow/psysh#67]
When using the built-in web server of PHP, psysh is executed when using `\Psy\Shell::debug(get_defined_vars());`, however the PHP execution is not resumed after quitting psysh (the browser hang waiting for an answer from the server). When launching a PHP script (with the web server), the PHP script is resumed after hitting ctrl-d in psysh.

I'm using the master branch of psysh on Ubuntu 14.04 with symfony2.

```
php --version

PHP 5.5.9-1ubuntu4 (cli) (built: Apr  9 2014 17:11:57) 
Copyright (c) 1997-2014 The PHP Group
Zend Engine v2.5.0, Copyright (c) 1998-2014 Zend Technologies
    with Zend OPcache v7.0.3, Copyright (c) 1999-2014, by Zend Technologies
```

**Comment by moolen** [comment]
+1

Im actually getting dropped into a second psysh shell but it takes about 60 seconds.

im running `PHP 5.5.9-1ubuntu4.4 (cli) (built: Sep  4 2014 06:56:34)`

The script to reproduce:
```PHP
<?php

require 'vendor/autoload.php';

$first = "foo";
\Psy\Shell::debug(get_defined_vars());

$second = "bar";
\Psy\Shell::debug(get_defined_vars());

echo $first . "\n";
echo $second . "\n";
```

It takes about 60s to get to the second shell.
About 60 seconds after i exited the second shell the output is transferred to the client, but the connection never gets closed.

**Comment by moolen** [comment]
a workaround: set the `default_socket_timeout` in your `/etc/php5/cli/php.ini` to 1 second.
so you have a lower latency between the breakpoints...

**Comment by bobthecow** [comment]
Oh, interesting. Thanks for that @moolen!

**Comment by oniric85** [comment]
+1 I've the same problem, is there any update on this? I can help if you need more information.

**Comment by ZalmanBlau** [comment]
+1

**Comment by ricpelo** [comment]
+1 here too.

**Comment by bobthecow** [comment]
Thanks to @5tevan, this is fixed in the development branch. It'll go out in the next release :)

**Comment by andreiglingeanu** [comment]
Amazing!

**Comment by bobthecow** [comment]
This should be fixed (as of v0.8.0). Let me know if you run into any issues.
